stories/index.tsx

```diff
@@ -61,6 +61,7 @@ import TextareaStory from './Input/TextareaStory';
 import TextFieldStory from './Input/TextFieldStory';
 import ToastStory from './Display/ToastStory';
 import TooltipStory from './Display/TooltipStory';
+import TypographyStory from '../src/General/Typography/TypographyStory';
```

Comment:
Just curious about the file path.
Seems to the other `Story.tsx` files are under `/storeies/General/`; 
like `import ProfilePictureStory from './General/ProfilePictureStory';`

Reply:
Yeah i'm trying to slowly migrate all the `Story` files to be in the same directory as their component haha. Think it makes more sense for them to all be in the same directory.